File: chapter1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
#include <stdlib.h>

int main(int argc, string argv[])
{

    int height = strtol(argv[1], NULL, 10);
    //printf("%d", height);
    (void)argc;

    for (int i = height; i > 0; i--)
    {
        for (int b = i - 1; b > 0 ; b--)

        {
            printf(" ");
        }

        for (int g = height - i; g >= 0; g--)
        {
        printf("#");


        }
        printf("  ");
        for (int g = height - i; g >= 0; g--)
        {
        printf("#");


        }

        printf("\n");
    }



}
// argc tells youhow many arguments,  argv is a list of strings that have the arguments in it.
//argv is a string, those imput swill not work as intended unless they wanted a string, otherwise they would need to convert it to what they want
//it maes things go faster.

File: chapter6/sentimental/mario.py
import cs50


while True:
    height = cs50.get_int ("Height: ")
    if height >= 0 and height <= 23:
        break

i = 1

while height > 0:
    print(" " * (height - 1), end="")
    print("#" * (i), end="")
    print("  ", end="")
    print("#" * (i), end="")
    print("")
    i += 1
    height -= 1



    # while b < 0:
    #     print(" ", end="")
    #     b -= 1
    # while g >= 0:
    #     print("#", end="")
    #     g -= 1

    # print("")
    # i -= 1


# for i in range(height - 1):
#     for b in range(i - 1):
#         print(" ", end="")
#         b -= 1
#     for g in range (g):
#         print("#", end="")
#         g -= 1
#     i -= 1